shopping-order-service/src/main/java/shopping/order/service/OrderServiceImpl.java

```diff
@@ -28,6 +28,22 @@ public OrderResponse createOrder(OrderRequest orderRequest) {
 		final Order order = repository.save(toOrderEntity(orderRequest, UUID.randomUUID().toString()));
 		return toOrderResponse(order);
 	}
+	@Override
+	public OrderResponse getOrderById(String id) {
+		final Order order = repository.findById(id).orElse(null);
```

Comment:
it should be null if the order couldn't be found in db. You may throw an exception in this case and then add a custom ExceptionHandler to handle that exception to response a proper error message to Client